src/Header.svelte

```diff
@@ -127,12 +128,26 @@
     top: 5rem;
   }
   .modal-register {
-    color: var(--color-foreground-6);
+    color: var(--color-foreground);
```

Comment:
This change changes the `Register` menu item in the mobile menu bar to be the same as in the desktop menu